src/driver/communication_interface.cc

```diff
@@ -359,6 +362,14 @@ bool CommunicationInterface::CanReceiveCommands(
   }
 }
 
+void CommunicationInterface::HandleCompliantPushReq(
+    const ::lcm::ReceiveBuffer*, const std::string&,
+    const robot_msgs::bool_t* msg) {
+  compliant_push_requested_ = msg->data;
+  new_plan_buffer_.plan = std::make_unique<PPType>();
+  new_plan_buffer_.utime = msg->utime;
+}
+
```

Comment:
Add handling in sim - instead of setting `compliant_push_requested_` we should just send a plan completion message with success:true

Reply:
I've removed the plan part of it, and now all it needs to do is set `compliant_push_active_` to true/false